workflow/controller/operator.go

```diff
@@ -1148,10 +1148,11 @@ func (woc *wfOperationCtx) podReconciliation(ctx context.Context) error {
 			// node is not a pod, it is already complete, or it can be re-run.
 			continue
 		}
+		recentlyStarted := recentlyStarted(node)
+		metrics.PodMissingMetric.WithLabelValues(strconv.FormatBool(recentlyStarted), string(node.Phase)).Add(0)
```

Comment:
Looks good. Can you just add a comment for it?

Reply:
@juliev0 Thank you for the review. I add the comment. :)

Reply:
that's weird - I'm not seeing it?

Reply:
You forgot to push your commits?

Reply:
I forgot to push. I pushed it.
@juliev0 @terrytangyuan